feature-libs/cart/import-export/components/export-entries/export-entries.service.ts

```diff
@@ -67,27 +65,16 @@ export class ExportEntriesService {
       : values?.toString() ?? '';
   }
 
-  protected get placement$(): Observable<string | undefined> {
-    return this.routingService
-      .getRouterState()
-      .pipe(map((route) => route.state?.semanticRoute));
-  }
-
-  protected getEntries(): Observable<OrderEntry[]> {
-    return this.placement$.pipe(
-      switchMap((placement) => {
-        switch (placement) {
-          case ExportCartRoutes.SAVED_CART_DETAILS: {
-            return this.getSavedCartEntries();
-          }
-          case ExportCartRoutes.CART:
-          default: {
-            return this.getActiveCartEntries();
-          }
-        }
-      }),
-      filter((entries) => entries?.length > 0)
-    );
+  protected getEntries(cartType: CartTypes): Observable<OrderEntry[]> {
+    switch (cartType) {
+      case CartTypes.EXISTING_SAVED_CART: {
+        return this.getSavedCartEntries();
+      }
```

Comment:
No need for change. But please let me explain how this implementation is different from what I meant initially.

My initial idea was to pass as an agument the `cartId` instead of the existing_saved_cart enum. Thanks to that, you would be able to export from an existing saved cart, regardless the route.

Please note that because of using `SavedCartDetailsService` behind the scenes, we still make indirect assumption that the route must contain `:savedCartId` param. Otherwise you aren't be able to export entries from the existing saved cart. 

This would be cumbersome e.g. when we (or customer) would like to add an `[Export to CSV]` button in a different page, e.g. in saved carts list page to add a new column with `[Export to CSV]` for every listed saved cart.

That said, if we _really don't have plans to export saved cart entries in different pages_, I accept the current design, to allow for working on other priorities.